src/clientStub.ts

```diff
@@ -328,7 +328,7 @@ export class DgraphClientStub {
       );
     }
 
-    private async callAPI<T>(path: string, config: { method?: string; body?: string; headers?: { [k: string]: string } }): Promise<T> {
+    public async callAPI<T>(path: string, config: { method?: string; body?: string; headers?: { [k: string]: string } }): Promise<T> {
```

Comment:
instead of exposing an internal method it's better to add `client.getHealth(<bool all>)` and `client.getState()` that way internal method isn't leaked, all alpha specific URL parts are hardcoded in one file, and no other classes depend on the `callAPI` method signature

Reply:
Sure, I'll do that. I just separated the cluster so that client wouldn't eventually be polluted with state/health functions.